overlay.d/05rhcos/usr/lib/systemd/system/rhcos-fix-root.service

```diff
@@ -0,0 +1,17 @@
+# Fixes the partition table / kargs from machines upgrading from
+# older releases before Ignition LUKS was available
+[Unit]
+Description=Fix root device
```

Comment:
How about "CoreOS Migrate Root LUKS Container"? (I'd call the service and script files something along those lines too to be a bit more descriptive.)